components/automate-ui/src/app/modules/infra-proxy/infra-search-bar/infra-search-bar.component.spec.ts

```diff
@@ -25,4 +25,17 @@ describe('InfraSearchBarComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('When a user toggles focus', () => {
+    // Toggling focus provides an active class to the form
+    // which gives our form an accessibilty ring
+    it('By default, the form should not be active', () => {
+      expect(component.formActive).toBe(false);
+    });
+
+    it('Should active the form when input is focused', () => {
+      component.toggleFocus();
+      expect(component.formActive).toBe(true);
+    });
+  });
```

Comment:
Author Note: All of our components should have more testing than simply "if" the component exists.  The more tests the better.